ruma-federation-api/src/query/get_profile_information/v1.rs

```diff
@@ -0,0 +1,45 @@
+//! [GET /_matrix/federation/v1/query/profile](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-query-profile)
+
+use ruma_api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "Get profile information, such as a display name or avatar, for a given user.",
+        name: "get_profile_information",
+        method: GET,
+        path: "/_matrix/federation/v1/query/profile",
+        rate_limited: false,
+        requires_authentication: true,
+    }
+
+    request: {
+        /// User ID to query.
+        #[ruma_api(query)]
+        pub user_id: String,
+        /// Profile field to query.
```

Comment:
Could you add a line break between fields? It helps for readability.

```suggestion
        pub user_id: String,
        
        /// Profile field to query.
```